docs/release-notes/usr-bin.rst

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**Improvements**
+
+-  Installation: ``.deb`` and ``.rpm`` Linux packages will now install master and agent binaries
+   into ``/usr/bin/`` instead of ``usr/local/bin/``.
```

Comment:
nit: I would offer an explanation why they changed, like "to be more in-line with Unix standards" or "with Debian and RedHat packaging standards" or something.  Or even just throw the phrase "more standard" in there somewhere.